### ISSUE TITLE
cc-env-var-create: provide visible labels

### ISSUE THREAD

**Issue by florian-sanders-cc** [CleverCloud/clever-components#559]
The `<cc-env-var-create>` component only relies on placeholders and provides no visible labels.
![image](https://user-images.githubusercontent.com/100240294/189308819-b985effe-5455-4073-93b8-c1d990556398.png)

For the moment, we added visually hidden labels to mitigate the issue.
We need to provide visible labels.
Considering this component is actually repeated for each variable, it may clutter the UI to provide a visible label for all of these.
We could rely on a common visible label like a "column header" in a table at the very top. 
The issue is that the visible label would then be very far from its input field and this is an a11y failure. Maybe this header could be fixed but i'm not sure it's an actual improvement. It may create more issues than it solves.

This one is tricky and I don't know how to handle this right now. Would need to think about it and check / test examples in the wild.

**Comment by hsablonniere** [comment]
We had a header before this new component but it won't work in the responsive mode.

**Comment by florian-sanders-cc** [comment]
> We had a header before this new component but it won't work in the responsive mode.

yes, fixed headers are always annoying in responsive mode (+ when users zoom in, and this is exactly for these users that we would want to make the label close to the input...)
